CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
         * `Session.fetch()` and `Session.fetch_into()` now takes additional parameters for associated attributes and add resulting record as part of the waveform info
+        * Rename `wfm` parameter to `waveform` in `fetch()` and `fetch_into()`
```

Comment:
niscope.Session.fetch() doesn't have a waveform parameter... I don't think it matters that the private stuff changed or not. It isn't user visible.

`fetch(num_samples=None, relative_to=niscope.FetchRelativeTo.PRETRIGGER, offset=0, record_number=0, num_records=None, timeout='datetime.timedelta(seconds=5.0)')`